c/windows.c

```diff
@@ -60,7 +60,7 @@ ptr S_ntdlerror(void) {
     char *s = s_ErrorStringImp(GetLastError(), "unable to load library");
     ret = Sstring_utf8(s, -1);
     free(s);
-    return s;
+    return ret;
```

Comment:
Good catch!

Reply:
Well somehow this wasn't added to the commit. As it is, it prints fixnums.